JS_Files/menu.js

```diff
@@ -0,0 +1,16 @@
+const navLinks = document.getElementById('navLinks');
+const showMenu = document.getElementById('menuBar');
+const hideMenu = document.getElementById('closeIcon');
+const closeMenu = document.getElementById('navItems');
+
+showMenu.onclick = function showMenu() {
```

Comment:
- [ ] On load of your page, the close button menu button is displayed as shown below. Kindly fix this.
![image](https://user-images.githubusercontent.com/94551484/219390947-0044e6d1-ce1a-4dfb-aac1-e7adf7e9376e.png)

Reply:
- [ ] This issue is well resolved :heavy_check_mark: